apps/codeforafrica/src/payload/fields/impact.js

```diff
@@ -0,0 +1,14 @@
+import { deepmerge } from "@mui/utils";
+
+const authors = (overrides) =>
+  deepmerge(
+    {
+      name: "impacts",
```

Comment:
1. Do `authors` have anything to do with `impact`?
2. Why is filename is `impact.js` but field name is `impacts`